modules/programs/zsh.nix

```diff
@@ -467,16 +475,6 @@ in
         default = [];
         example = literalExpression ''
           [
-            {
-              # will source zsh-autosuggestions.plugin.zsh
-              name = "zsh-autosuggestions";
-              src = pkgs.fetchFromGitHub {
-                owner = "zsh-users";
-                repo = "zsh-autosuggestions";
-                rev = "v0.4.0";
-                sha256 = "0z6i9wjjklb4lvr7zjhbphibsyx51psv50gm07mbb0kj9058j6kc";
-              };
-            }
```

Comment:
Replacing this with `wd` since [`zsh.autosuggestion.enable`](https://nix-community.github.io/home-manager/options.xhtml#opt-programs.zsh.autosuggestion.enable) is now an option and I couldn't quickly find another plugin with completions. Open to swapping this with something else if `wd` isn't preferred though!